src/app/technical-record/vehicle-summary/vehicle-summary.component.html

```diff
@@ -82,54 +85,72 @@
   </div>
   <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'trl'">
     <dt class="govuk-summary-list__key">
-      Load sensing valve
+      Suspension type
     </dt>
-    <dd id="test-loadSensingValve" class="govuk-summary-list__value">
-      <div *ngIf="!!activeRecord.brakes; else hyphen">
-        {{ activeRecord.brakes.loadSensingValve | DefaultNullOrEmpty }}
-      </div>
+    <dd id="test-suspensionType" class="govuk-summary-list__value">
+      {{ activeRecord.suspensionType | DefaultNullOrEmpty }}
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'trl'">
+  <div class="govuk-summary-list__row">
     <dt class="govuk-summary-list__key">
-      Antilock braking system
+      Road friendly suspension
     </dt>
-    <dd id="test-antilockBrakingSystem" class="govuk-summary-list__value">
-      <div *ngIf="!(activeRecord.brakes == null); else hyphen">
-        {{ activeRecord.brakes.antilockBrakingSystem | DefaultNullOrEmpty }}
-      </div>
+    <dd id="test-roadFriendly" class="govuk-summary-list__value">
+      {{ activeRecord.roadFriendly | DefaultNullOrEmpty }}
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'trl'">
+  <div class="govuk-summary-list__row">
     <dt class="govuk-summary-list__key">
-      Brake actuator
+      Vehicle class
     </dt>
-    <dd id="test-brakeActuator" class="govuk-summary-list__value">
-      {{ activeRecord.axles.slice(-1)[0].brakes.brakeActuator | DefaultNullOrEmpty }}
+    <dd id="test-vehicleClassDescription" class="govuk-summary-list__value">
+      {{
+        techRecHelpers.isNullOrEmpty(activeRecord.vehicleClass)
```

Comment:
we should stop using this service. Use instead `!!activeRecord.vehicleClass` or just `Object.keys(activeRecord.vehicleClass)`. We would reduce code footprint not using that service

Reply:
It is probably best if we extract that into a function (or Pipe) and do not do inline js so it becomes also easily testable

Reply:
moved to a function.